### PR TITLE
BASE: Allow options to come after the command on the command line

### PR DESCRIPTION
**Pull request by bgK**

### CODE REVIEW

**Comment by sev-** [comment]
It is not clear what exactly it does. Could you please elaborate with examples?

**Comment by bgK** [comment]
@criezy Good catch, I fixed `--list-saves` to behave more like the other commands with regards to entering multiple commands. The rest of your proposed changes sound right but are, I think, out of the scope of this pull request.

File: base/commandLine.cpp [hunk 1]
```diff
@@ -455,7 +458,7 @@ Common::String parseCommandLine(Common::StringMap &settings, int argc, const cha
 				// FIXME: Need to document this.
 				// TODO: Make the argument optional. If no argument is given, list all saved games
 				// for all configured targets.
-				return "list-saves";
```
Comment (criezy) [inline base/commandLine.cpp]: For consistency you should probably check that `command` is empty before setting it here and if it is not display an error as done in the DO_COMMAND.

**Comment by bgK** [comment]
Thanks for your reviews, I merged this manually.

**Pull request closed without merging**